style.css

```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  background-image: linear-gradient(
+      115deg,
+      rgba(58, 58, 158, 0.8),
+      rgba(136, 136, 206, 0.7)
+    ),
+    url("https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg");
+  width: 100%;
+  height: auto;
+  background-repeat: no-repeat;
+  font-size: 1.2vw;
+  color: aliceblue;
+}
+#head {
+  max-width: 750px;
+  margin: 4rem auto;
+  padding: 0.6rem;
+}
+#main {
```

Comment:
For clean codes🚀, i could suggest that you use empty space after every `}` for good practice and to avoid linter errors